### DISCUSSION TITLE
doesn't work at all !!

### DISCUSSION THREAD

**Discussion by c4ir0** [projectdiscovery/httpx#774] [Q&A] [answered]
It doesnt sem to work as expected in https://github.com/projectdiscovery/httpx#url-probe
or anything else !!
can't get the version 
I installed it as mentioned in the man
pls help 

![Capture](https://user-images.githubusercontent.com/108268587/190874781-414a227e-f172-4a9b-9c5b-9a38886a7eed.JPG)

**Comment by forgedhallpass** [comment, marked as answer]
That is because you are not using our `httpx`. See below:

![image](https://user-images.githubusercontent.com/13679401/190896876-b789b5f7-69f9-4b20-b1bc-4f64429c2426.png)

**Reply by c4ir0** [reply]
yes I realised it's another tool with the same name, despite I installed UR tool with the way u provided, weird

**Reply by forgedhallpass** [reply]
The python version is included in kali by default.

Related discussion: https://github.com/projectdiscovery/httpx/discussions/775#discussioncomment-3673723

**Reply by zoutre** [reply]
@forgedhallpass Thank you for the update - issue is the same. Also appears there is an alternative install for Kali as httpx-toolkit:
sudo apt install httpx-toolkit

Please confirm whether httpx-toolkit == httpx in this repo; If yes, please share uninstall recommendation to remove the go installation, so that it may be replaced with httpx-toolkit on Kali.

Thank you in advance

**Reply by forgedhallpass** [reply]
@zoutre the issue cannot be the same if you understood what I was saying :)
I've listed the potential steps one can take in the previous link https://github.com/projectdiscovery/httpx/discussions/775#discussioncomment-3673723.

If you've installed our `httpx` using `go install ...` then the binary will be under `$GOPATH/bin/httpx`. If you want to "uninstall" it, you just need to delete it from there.

[`httpx-toolkit`](https://www.kali.org/tools/httpx-toolkit) based on the link indeed points to our repository, but it seems to contain version `1.1.5`, whereas the latest version currently is `1.2.4`.

**Reply by zoutre** [reply]
@forgedhallpass Simply meant that the the python version is installed and the path to the go version was correct; so, was able to validate both tools do exist on my system in different paths. :)

Perfect, version is not that important; will remove and install with apt. 

Thanks!
